scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/TimeRangeExpression.java

```diff
@@ -48,7 +48,8 @@ public Windows computeRange(final SimulationResults simulationResults, final Pla
       if(minTimepoint.isPresent() && maxTimepoint.isPresent()) {
         final var anchorActSearch = new ActivityExpression.Builder()
             .basedOn(actTemplate)
-            .startsIn(inter).build(); //check if it exists IN the windows, not just the upper and lower bounds of the interval
+            //.startsIn(inter)
```

Comment:
Minor: commented line

Reply:
Yeah, it's not worth it to update it because this class is going to be deleted soon.